2025-03-28 15:38:18,280 INFO : Microblog startup [in /Users/asanbeksamudin/Documents/projects/mine/python/flask/microblog/microblog/app/__init__.py:40]
2025-03-31 08:34:46,256 INFO : Microblog startup [in /Users/asanbeksamudin/Documents/projects/mine/python/flask/microblog/microblog/app/__init__.py:40]
2025-03-31 08:34:59,282 ERROR : Exception on /index [GET] [in /Users/asanbeksamudin/Documents/projects/mine/python/flask/microblog/venv/lib/python3.12/site-packages/flask/app.py:1414]
Traceback (most recent call last):
  File "/Users/asanbeksamudin/Documents/projects/mine/python/flask/microblog/venv/lib/python3.12/site-packages/flask/app.py", line 2190, in wsgi_app
    response = self.full_dispatch_request()
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/asanbeksamudin/Documents/projects/mine/python/flask/microblog/venv/lib/python3.12/site-packages/flask/app.py", line 1486, in full_dispatch_request
    rv = self.handle_user_exception(e)
         ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/asanbeksamudin/Documents/projects/mine/python/flask/microblog/venv/lib/python3.12/site-packages/flask/app.py", line 1484, in full_dispatch_request
    rv = self.dispatch_request()
         ^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/asanbeksamudin/Documents/projects/mine/python/flask/microblog/venv/lib/python3.12/site-packages/flask/app.py", line 1469, in dispatch_request
    return self.ensure_sync(self.view_functions[rule.endpoint])(**view_args)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/asanbeksamudin/Documents/projects/mine/python/flask/microblog/venv/lib/python3.12/site-packages/flask_login/utils.py", line 290, in decorated_view
    return current_app.ensure_sync(func)(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/asanbeksamudin/Documents/projects/mine/python/flask/microblog/microblog/app/routes.py", line 22, in index
    posts = current_user.followed_posts().paginate(page, app.config['POSTS_PER_PAGE'], False)
            ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
TypeError: Query.paginate() takes 1 positional argument but 4 were given
2025-03-31 08:35:01,288 ERROR : Exception on /index [GET] [in /Users/asanbeksamudin/Documents/projects/mine/python/flask/microblog/venv/lib/python3.12/site-packages/flask/app.py:1414]
Traceback (most recent call last):
  File "/Users/asanbeksamudin/Documents/projects/mine/python/flask/microblog/venv/lib/python3.12/site-packages/flask/app.py", line 2190, in wsgi_app
    response = self.full_dispatch_request()
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/asanbeksamudin/Documents/projects/mine/python/flask/microblog/venv/lib/python3.12/site-packages/flask/app.py", line 1486, in full_dispatch_request
    rv = self.handle_user_exception(e)
         ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/asanbeksamudin/Documents/projects/mine/python/flask/microblog/venv/lib/python3.12/site-packages/flask/app.py", line 1484, in full_dispatch_request
    rv = self.dispatch_request()
         ^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/asanbeksamudin/Documents/projects/mine/python/flask/microblog/venv/lib/python3.12/site-packages/flask/app.py", line 1469, in dispatch_request
    return self.ensure_sync(self.view_functions[rule.endpoint])(**view_args)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/asanbeksamudin/Documents/projects/mine/python/flask/microblog/venv/lib/python3.12/site-packages/flask_login/utils.py", line 290, in decorated_view
    return current_app.ensure_sync(func)(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/asanbeksamudin/Documents/projects/mine/python/flask/microblog/microblog/app/routes.py", line 22, in index
    posts = current_user.followed_posts().paginate(page, app.config['POSTS_PER_PAGE'], False)
            ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
TypeError: Query.paginate() takes 1 positional argument but 4 were given
2025-03-31 08:35:02,117 ERROR : Exception on /index [GET] [in /Users/asanbeksamudin/Documents/projects/mine/python/flask/microblog/venv/lib/python3.12/site-packages/flask/app.py:1414]
Traceback (most recent call last):
  File "/Users/asanbeksamudin/Documents/projects/mine/python/flask/microblog/venv/lib/python3.12/site-packages/flask/app.py", line 2190, in wsgi_app
    response = self.full_dispatch_request()
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/asanbeksamudin/Documents/projects/mine/python/flask/microblog/venv/lib/python3.12/site-packages/flask/app.py", line 1486, in full_dispatch_request
    rv = self.handle_user_exception(e)
         ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/asanbeksamudin/Documents/projects/mine/python/flask/microblog/venv/lib/python3.12/site-packages/flask/app.py", line 1484, in full_dispatch_request
    rv = self.dispatch_request()
         ^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/asanbeksamudin/Documents/projects/mine/python/flask/microblog/venv/lib/python3.12/site-packages/flask/app.py", line 1469, in dispatch_request
    return self.ensure_sync(self.view_functions[rule.endpoint])(**view_args)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/asanbeksamudin/Documents/projects/mine/python/flask/microblog/venv/lib/python3.12/site-packages/flask_login/utils.py", line 290, in decorated_view
    return current_app.ensure_sync(func)(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/asanbeksamudin/Documents/projects/mine/python/flask/microblog/microblog/app/routes.py", line 22, in index
    posts = current_user.followed_posts().paginate(page, app.config['POSTS_PER_PAGE'], False)
            ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
TypeError: Query.paginate() takes 1 positional argument but 4 were given
2025-03-31 08:38:54,313 INFO : Microblog startup [in /Users/asanbeksamudin/Documents/projects/mine/python/flask/microblog/microblog/app/__init__.py:40]
2025-03-31 08:38:57,460 ERROR : Exception on /index [GET] [in /Users/asanbeksamudin/Documents/projects/mine/python/flask/microblog/venv/lib/python3.12/site-packages/flask/app.py:1414]
Traceback (most recent call last):
  File "/Users/asanbeksamudin/Documents/projects/mine/python/flask/microblog/venv/lib/python3.12/site-packages/flask/app.py", line 2190, in wsgi_app
    response = self.full_dispatch_request()
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/asanbeksamudin/Documents/projects/mine/python/flask/microblog/venv/lib/python3.12/site-packages/flask/app.py", line 1486, in full_dispatch_request
    rv = self.handle_user_exception(e)
         ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/asanbeksamudin/Documents/projects/mine/python/flask/microblog/venv/lib/python3.12/site-packages/flask/app.py", line 1484, in full_dispatch_request
    rv = self.dispatch_request()
         ^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/asanbeksamudin/Documents/projects/mine/python/flask/microblog/venv/lib/python3.12/site-packages/flask/app.py", line 1469, in dispatch_request
    return self.ensure_sync(self.view_functions[rule.endpoint])(**view_args)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/asanbeksamudin/Documents/projects/mine/python/flask/microblog/venv/lib/python3.12/site-packages/flask_login/utils.py", line 290, in decorated_view
    return current_app.ensure_sync(func)(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/asanbeksamudin/Documents/projects/mine/python/flask/microblog/microblog/app/routes.py", line 22, in index
    posts = current_user.followed_posts().paginate(page, app.config['POSTS_PER_PAGE'], False)
            ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
TypeError: Query.paginate() takes 1 positional argument but 4 were given
2025-03-31 08:40:25,715 INFO : Microblog startup [in /Users/asanbeksamudin/Documents/projects/mine/python/flask/microblog/microblog/app/__init__.py:40]
2025-03-31 09:35:16,576 INFO : Microblog startup [in /Users/asanbeksamudin/Documents/projects/mine/python/flask/microblog/microblog/app/__init__.py:40]
2025-04-01 07:49:20,486 INFO : Microblog startup [in /Users/asanbeksamudin/Documents/projects/mine/python/flask/microblog/microblog/app/__init__.py:44]
